README.md

```diff
@@ -6,42 +6,48 @@
 
 This is a Chainlink plugin implementation for version `4.x` of [web3.js](https://github.com/web3/web3.js).
 
-###### Get it from the NPM Registry
+## Prerequisites
+
+-   :gear: [NodeJS](https://nodejs.org/) (LTS/Fermium)
+-   :toolbox: [Yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/package/npm)
+
+## Installation
+
+You can install the package either using [npm](https://www.npmjs.com/package/web3) or using [Yarn](https://yarnpkg.com/package/web3)
+
+### Using npm
 
 ```bash
-yarn add web3-plugin-chainlink
+npm install web3
 ```
 
-## Prerequisites
+### Using Yarn
 
--   :gear: [NodeJS](https://nodejs.org/) (LTS/Fermium)
--   :toolbox: [Yarn](https://yarnpkg.com/)
+```bash
+yarn add web3
+```
 
-## Getting Started
+## Found an issue or have a question or suggestion
 
--   :writing_hand: If you have questions [submit an issue](https://github.com/web3/web3.js/issues/new) or join us on [Discord](https://discord.gg/yjyvFRP)
+-   :writing_hand: If you found an issue or have a question or suggestion [submit an issue](https://github.com/ChainSafe/web3.js-plugin-chainlink/issues/new) or join us on [Discord](https://discord.gg/yjyvFRP)
     ![Discord](https://img.shields.io/discord/593655374469660673.svg?label=Discord&logo=discord)
 
-After cloning the repo,
+## Run the tests
+
+You may like to run the tests and examin their code to see how to use the plugin. And this would be also useful if you are a plugin writer.
 
--   Run `yarn`
+First clone the repo locally. And then:
+
+-   Run `yarn` or `npm i`
     -   Installs dependencies and builds the plugin
--   Run `yarn test`
+-   Run `yarn test` or `npm test`
     -   Runs the [unit test](https://github.com/ChainSafe/web3.js-plugin-chainlink/blob/master/test/unit/plugin.test.ts) that instantiates an instance of `Web3`, configures it with a provider, then registers the [ChainlinkPlugin](https://github.com/ChainSafe/web3.js-plugin-chainlink/blob/master/src/index.ts) on the `Web3` instance
         -   `ChainlinkPlugin` takes an `AggregatorV3InterfaceABI` as the first argument, and the deployed contract address as the second
 
-```typescript
```

Comment:
any reason why the example is removed?

Reply:
Sorry, this seems to be un-updated and not a complete part of the code of the written plugin and not an example of  using it: https://github.dev/ChainSafe/web3.js-plugin-chainlink/blob/620ec464385f9eee0eae43d1e37a41db0ddc7cf7/src/index.ts#L10

Reply:
It would be better to update the example than to explain how to use it, no?

Reply:
That is right, we will add an article on how to use this plugin and it would contain examples for js and ts. This would be part of https://github.com/web3/web3.js/issues/5569

And we would be able to write an example once the discussion on this is concluded:
https://github.com/ChainSafe/web3.js-plugin-chainlink/issues/1 As for now without doing something about how web3 is needed to be augmented, the example would not be clear, I think. Or what do you think? Should I write an example on the as-is for the module augmentation? Or leave it to later till we conclude a better way? 
And by the way, I am currently trying to use the plugin in a local test project to see what could be the best way to do the module augmentation. But I think that we can merge this MR independently...
Or what do you say?

Reply:
> That is right, we will add an article on how to use this plugin and it would contain examples for js and ts. This would be part of https://github.com/web3/web3.js/issues/5569

I don't think we are allowed to create documentation for this in web3.js, otherwise, we would need to allow everyone to create docs for their plugins there.

> And by the way, I am currently trying to use the plugin in a local test project to see what could be the best way to do the module augmentation. But I think that we can merge this MR independently...
Or what do you say?

I'm fine with merging PR as is if it's not easy to write example how to use it :)

Reply:
Yes, that article should not be part of web3.js repo. So it will be either as part of this repository (in a .md file for example, and this is what I think is the best). Or away from there as something on medium.com, for example.

And so I hope you approve this MR, please.
Thanks,

Reply:
Oh sorry, did not know you were waiting for my approval. I'm avoiding requesting changes just so I'm not a blocker to you guys 😅